java/libs/src/main/java/org/physical_web/collection/PwsClient.java

```diff
@@ -61,8 +78,42 @@ public void setPwsEndpoint(String pwsEndpoint) {
     mPwsEndpoint = pwsEndpoint;
   }
 
-  private String constructPwsUrl(String path) {
-    return mPwsEndpoint + "/" + path;
+  /**
+   * Set the API key the PWS endpoint uses.
+   * @param pwsApiKey The new PWS endpoint.
+   */
+  public void setPwsApiKey(String pwsApiKey){
+      apiKey = pwsApiKey;
+  }
+
+  /**
+   * Set the API version the PWS endpoint uses.
+   * @param pwsApiVersion The new PWS endpoint.
+   */
+  public void setPwsApiVersion(int pwsApiVersion){
```

Comment:
I think it'd be better if we didn't have this method...just add a second param to setEndpoint.